### ISSUE TITLE
Column flow_metadatafolder referenced in DIMon_Statistics job

### ISSUE THREAD

**Issue by hlugt** [eomnl/sas-jobmon#19]
Hi there B and others, doing a fresh install from scratch. But when importing the latest SASBatch package containing the ETL jobs, a warning is given on a referenced column, not existent in latest (nor previous) version of table.

> WARN  - The table "DIMON_FLOWS" did not have a matching column named "FLOW_METADATAFOLDER".

This also leads to warnings when executing the job:

>  Line 1,187: WARNING: Length of character variable interpolated_ind has already been set. Use the LENGTH statement as the very first statement in the DATA STEP to declare the length of a character variable.)

We have reverted to a previous version of the ETL job, but maybe it is something to look into?

Regards, HL.

**Comment by bheinsius** [comment]
You are correct. It does not affect the working of the product though, will fix when i have time.
If you need further help installing the product, let me know.
